test/clean-local-registry-tag.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
It wasn't clear to me how this was cleaning the registry at first. It would be good to include a comment here explaining that we're using busybox as the the "deleted" image.
Still, why can't we just delete the tag?

Reply:
Couldn't find a way to remove tag from remote repo (we can delete the tag from local docker cache, but not from docker registry running on the node). Alternatively we can delete registry container and just restart it

Reply:
Hmm, but we can't delete the registry since it'll be shared between parallel jobs. With that in mind, don't we restart the registry for every job? or is it once per node-startup?

Reply:
Running registry is part of our node provisioning script.

Reply:
Cool cool. Maybe just a comment then? explaining when to use this and what it does.